draft-ietf-core-new-block.xml

```diff
@@ -1975,6 +1980,60 @@ o  Reference: [RFCXXXX]]]></artwork>
         new transmission of the payloads that have not been acknowledged under
         these adverse traffic conditions.</t>
 
+        <t>If there is likely to be the possibility of network transient
+        losses, then the use of NON should be considered.</t>
+      </section>
+    </section>
+    <section anchor="REL" title="Examples with Reliable Transports">
+      <t>The notations provided in <xref target="legend"></xref> are used in
+      the following subsections.</t>
+
+      <section title="Q-Block1 Option">
+        <t>Let's now consider the use Q-Block1 Option with reliable transports
+        as shown in <xref target="rel3"></xref>. There is no acknowledgment of
+        packets at the CoAP layer, just the final result.</t>
+
+        <t><figure anchor="rel3"
+            title="Example of reliable Request with Q-Block1 Option">
+            <artwork><![CDATA[        CoAP        CoAP
+       Client      Server
+         |          |
+         +--------->| PUT /path T:0xf0 RT=10 QB1:0/1/1024
+         +--------->| PUT /path T:0xf1 RT=10 QB1:1/1/1024
+         +--------->| PUT /path T:0xf2 RT=10 QB1:2/1/1024
+         +--------->| PUT /path T:0xf3 RT=10 QB1:3/0/1024
+         |<---------+ 2.04
+         |          |
+]]></artwork>
+          </figure></t>
+
+        <t>If there is likely to be the possibility of network transient
+        losses, then the use of NON should be considered.</t>
```

Comment:
```suggestion
        losses, then the use of unrelible transport with NON should be considered.</t>
```

As this is an example, I wonder whether there is a value in having this sentence

Reply:
Updated